plugins/test/runway.js

```diff
@@ -93,15 +93,13 @@ describe('TranslationServer', function () {
             path: '/translateTo'
         });
 
-        console.log(tds_xml);
+        // console.log(tds_xml);
         
         xml = parser.parseFromString(tds_xml);
-        gj = osmtogeojson(xml);
```

Comment:
`osmtogeojson` call doesn't like ways that are missing all of their nodes like the example data and completely fails to convert.  So instead of converting it, use XML below to make sure that the translation happened correctly.

Reply:
Very true, and nice workaround.